src/utils/node/dynamicProps.js

```diff
@@ -24,7 +24,7 @@ export async function setCustomPropTypes(isVuelessEnv) {
     const customProps =
       componentName in vuelessConfig.component && vuelessConfig.component[componentName].props;
 
-    if (customProps) {
+    if (customProps && vuelessConfig.component[componentName].storybook !== false) {
```

Comment:
if `componentName` is not set it `vuelessConfig` may occurs error `vuelessConfig.component[componentName].storybook`

```js
const isHiddenStories =
      componentName in vuelessConfig.component &&
      vuelessConfig.component[componentName].storybook === false;

if (customProps && !isHiddenStories) {
```